### PR TITLE
40 hours is missing for all developers.

### PR DESCRIPTION
**Pull request by AliHanifdev1**

### CODE REVIEW

**Review by MKRNaqeebi — CHANGES_REQUESTED** [review]
remove console.log
name function and variable properly

File: src/components/FortyHours.vue [hunk 1]
```diff
@@ -86,28 +95,49 @@ export default {
         });
       });
     },
-    fetchIssues() {
+    allI(repos) {
```
Comment (MKRNaqeebi) [inline src/components/FortyHours.vue]: name this function properly

File: src/components/FortyHours.vue [hunk 2]
```diff
@@ -86,28 +95,49 @@ export default {
...
           },
         })
           .then((response) => {
+            this.allI(response.data);
             if (response.data.length < 1) {
               nextPage = false;
               return;
             }
-            this.formateIssue(response.data);
+            //this.formate(response.data);
```
Comment (MKRNaqeebi) [inline src/components/FortyHours.vue]: remove this unnecessary comment

**Review by MKRNaqeebi — CHANGES_REQUESTED** [review]

File: src/components/FortyHours.vue (hunk 2 above)
Comment (AliHanifdev1) [reply src/components/FortyHours.vue]: done

File: src/components/FortyHours.vue (hunk 1 above)
Comment (AliHanifdev1) [reply src/components/FortyHours.vue]: done

File: src/components/FortyHours.vue [hunk 3]
```diff
@@ -86,28 +114,53 @@ export default {
         });
       });
     },
-    fetchIssues() {
+    AllIssues(repos) {
```
Comment (MKRNaqeebi) [inline src/components/FortyHours.vue]: make the camel case's 1st letter small.

File: src/components/FortyHours.vue [hunk 4]
```diff
@@ -86,28 +114,53 @@ export default {
...
+        axios({
+          method: "get",
+          url: `https://api.github.com/repos/${repo.full_name}/issues?state=closed&archived:false+closed=${startingDay}`,
+          auth: {
+            username: this.username,
+            password: this.password,
+          },
+        })
+          .then((response) => {
+            //console.log("res", response.data);
```
Comment (MKRNaqeebi) [inline src/components/FortyHours.vue]: remove it

**Review by MKRNaqeebi — CHANGES_REQUESTED** [review]

File: src/components/FortyHours.vue (hunk 3 above)
Comment (AliHanifdev1) [reply src/components/FortyHours.vue]: done

File: src/components/FortyHours.vue (hunk 4 above)
Comment (AliHanifdev1) [reply src/components/FortyHours.vue]: done

**Review by MKRNaqeebi — APPROVED** [review]

**Pull request merged**